src/upload-tests.proj

```diff
@@ -77,6 +77,12 @@
     <HelixArchLabel>$(ArchGroup)</HelixArchLabel>
     <HelixConfigLabel>$(ConfigurationGroup)</HelixConfigLabel>
     <MaxRetryCount>1</MaxRetryCount>
+    <!-- Unique container name per build and configuration, e.g., corefx-beta-12345-01-windows-x64 -->
+    <!-- Azure Storage containers can only contain [a-z, 0-9, -] in their names and $(OSGroup) is "Windows_NT" on Windows... -->
+    <ProcessedOSGroup>$(OSGroup.ToLower())</ProcessedOSGroup>
+    <ProcessedOSGroup Condition="'$(TargetsWindows)' == 'true'">windows</ProcessedOSGroup>
+    <ProcessedArchGroup>$(ArchGroup.ToLower())</ProcessedArchGroup>
+    <ContainerName Condition="'$(ContainerName)' == '' And '$(IsOfficial)' != ''">corefx-$(PreReleaseLabel)-$(BuildNumberMajor)-$(BuildNumberMinor)-$(ProcessedOSGroup)-$(ProcessedArchGroup)-$(TargetGroup)</ContainerName>
```

Comment:
I think we should use the same naming convention for official builds so I'd suggest you just add a qualifier for official vs ci builds. There is also potentially a need to include the branch name because 2 different branches may end up building with the same build information.